### PR TITLE
change default of COMMON_ENABLE_BASIC_AUTH to False

### PR DESCRIPTION
**Pull request by dhpollack**
I changed the default behavior of basic authorization to false.  I found the default behavior unintuitive.  Upon installation of the default image via vagrant, going to the server from a remote host (including the vagrant host computer) prompts the user for a username/password.  The default behavior should be that the edx lms/cms "just work" without a username/password, right?  Furthermore, the documentation does not mention that the username/password is edx/edx, and one can only find this information by going into the source files.  Finally, I didn't see a great reason to use basic auth at all, and thus if someone wants to use it, they probably should explicitly enable it, instead of this being the default behavior.

### CODE REVIEW

**Comment by jarv** [comment]
Yeah this change came recently when we did a refactor of the vars for basic auth. I'm fine with making basic auth turned off as the default if the rest of the team is ok with it.

**Comment by jarv** [comment]
We should probably merge this into the release branch as well if we take it

**Comment by dhpollack** [comment]
@jarv thanks for the quick reply.  Let me know if I need to anything else.  I have already signed the edx waiver form and added myself to the AUTHORS file for previous pull requests.

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -131,7 +131,7 @@ EOF
 if [[ $basic_auth == "true" ]]; then
     # vars specific to provisioning added to $extra-vars
     cat << EOF_AUTH >> $extra_vars_file
-COMMON_ENABLE_BASIC_AUTH: True
+COMMON_ENABLE_BASIC_AUTH: False
```
Comment (hkim823) [inline util/jenkins/ansible-provision.sh:134]: I don't think this should be set to False, this script is used to provision developer sandboxes, and should be set to True in this scenario.

Comment (jarv) [inline util/jenkins/ansible-provision.sh:134]: @hkim823 good catch, I only saw the first diff.
@dhpollack can you resubmit this pull to our release branch with only the first change to defaults/main.yml?

Comment (dhpollack) [inline util/jenkins/ansible-provision.sh:134]: alright, see pull #1286

**Comment by hkim823** [comment]
Closing this PR due to new PR

**Comment by dhpollack** [comment]
@hkim823 got it.  Would it make sense then to remove the same ppas from [this file](https://github.com/edx/edx-platform/blob/master/requirements/system/ubuntu/apt-repos.txt)?  Or perhaps that entire folder, what got me looking into all of this was the existence of these supposed "requirements".

**Pull request closed without merging**